### ISSUE TITLE
Renaming scrollbar widget label

### ISSUE THREAD

**Issue by denglert** [ioam/holoviews#1160]
Although one can rename the axis labels by 
~~~~
all_ds = all_ds.redim(cba=('cba', r'cos(b-a)'), tb=('tb', 'tan(b)'), mA_bin=('mA_bin', 'm_A [GeV]'))
~~~~
the label is not changed accordingly when a scrollbar is created. E.g.:
~~~~
a  = all_ds.to(hv.Points, kdims=['cba', 'tb'], vdims=[], groupby=['mA_bin'])
~~~~
still shows `mA_bin` not `m_A [GeV]`.

**Comment by philippjfr** [comment]
Thanks, adding a ``Dimension.label`` is fairly recent so the sliders aren't using it yet.

**Comment by philippjfr** [comment]
Fixed in #1205.
